runhouse/resources/hardware/cluster.py

```diff
@@ -1082,47 +1062,22 @@ def _start_or_restart_helper(
         resync_rh: Optional[bool] = None,
         restart_ray: bool = True,
         restart_proxy: bool = False,
+        parallel: bool = True,
     ):
         if not self.is_up():
             raise ConnectionError(
                 f"Could not reach {self.name} {self.head_ip}. Is cluster up?"
             )
 
-        image_secrets, image_env_vars = self._sync_image_to_cluster()
-
-        # If resync_rh is not explicitly False, check if Runhouse is installed editable
-        local_rh_package_path = None
-        if resync_rh is not False:
-            local_rh_package_path = Path(
-                importlib.util.find_spec("runhouse").origin
-            ).parent
-
-            installed_editable_locally = (
-                not _rh_install_url
-                and local_rh_package_path.parent.name == "runhouse"
-                and (local_rh_package_path.parent / "setup.py").exists()
-            )
-
-            if installed_editable_locally:
-                logger.debug("Runhouse is installed locally in editable mode.")
-                resync_rh = True
-            else:
-                # We only want this to be set if it was installed editable locally
-                local_rh_package_path = None
+        if resync_rh:
```

Comment:
do we not need to keep the `if rsync_rh is not None` to do detection of if we need to sync a local runhouse?

Reply:
We need to add a force argument here which takes in resync_rh, and if local runhouse package is editable then force=True by default (otherwise false by default).

Reply:
`force_sync_local`